src/main/java/simpli/tasks/Event.java

```diff
@@ -0,0 +1,39 @@
+package simpli.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
```

Comment:
Could we consider enhancing the readability of the DateTimeFormatter pattern in the Event class? Perhaps using a more explicit pattern like dd/MM/yyyy HH:mm